sigle/src/modules/publicStory/PublicStory.tsx

```diff
@@ -90,6 +115,7 @@ export const PublicStory = ({ story, settings }: PublicStoryProps) => {
               css={{
                 marginLeft: '-$20',
                 marginRight: '-$20',
+                width: '100%',
```

Comment:
what was the reason to add this one? Adding it make the negative margin not work properly

Reply:
yes so after speaking with Quentin regarding the design, he wanted the max width of the images to be the size of the content width, so that's why the full width property is there. I can check back with him regarding this? :)